### PR TITLE
`@mycrypto/wallets@1.5.1`

### PR DESCRIPTION
**Pull request by FrederikBolding**
Update `@mycrypto/wallets` to version `1.5.1`

### CODE REVIEW

**Review by Mrtenz — APPROVED** [review]

**Review by Austindgk23 — COMMENTED** [review]
Ok

**Review by farisei77 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "@mycrypto/eth-scan": "3.4.4",
```
Comment (djs19901) [inline package.json:28]: Unlock my crypto now

File: package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
...
     "@mycrypto/ui": "0.24.1",
     "@mycrypto/unlock-scan": "1.2.0",
-    "@mycrypto/wallets": "1.5.0",
+    "@mycrypto/wallets": "1.5.1",
```
Comment (djs19901) [inline package.json:31]: bc1qd0paklsxm8xh4ml6zyuxdm9sn7flt0aezd02l3

File: yarn.lock [hunk 3]
```diff
@@ -3032,10 +3032,10 @@
...
     "@findeth/abi" "^0.7.1"
 
-"@mycrypto/wallets@1.5.0":
-  version "1.5.0"
-  resolved "https://registry.yarnpkg.com/@mycrypto/wallets/-/wallets-1.5.0.tgz#09a57cfc61ef448aac7169cd1ebe2ecd74c7c885"
-  integrity sha512-DQxYoJDamiKLszbRchfuDqYsl2gfG/PKxQ098GX2AGf5SktYaS6xDqkTL/R9REOYmq5alNQFf3PBK6MXEIHI1A==
+"@mycrypto/wallets@1.5.1":
+  version "1.5.1"
+  resolved "https://registry.yarnpkg.com/@mycrypto/wallets/-/wallets-1.5.1.tgz#f7afeb66669476fdb04e088f95edd77dbf7d5492"
+  integrity sha512-O27Zv9yEbFWnSoY7yweMQiWkzSFO5Aj8pKvCMFGS0z/uyMfpIGPdMmtTFr8lA/I8N21qQfGnHCRXw+wQ2eRDZg==
```
Comment (djs19901) [inline yarn.lock:3038]: My btc wallet bc1qd0paklsxm8xh4ml6zyuxdm9sn7flt0aezd02l3

**Review by tiwtiw11 — APPROVED** [review]

**Pull request merged**